pkgs/applications/graphics/xournalpp/default.nix

```diff
@@ -23,18 +24,20 @@
 
 stdenv.mkDerivation rec {
   pname = "xournalpp";
-  version = "1.1.3";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "xournalpp";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-Hn7IDnbrmK3V+iz8UqdmHRV2TS4MwYSgYtnH6igbGJ8=";
+    sha256 = "sha256-0xsNfnKdGl34qeN0KZbII9w6PzC1HvvO7mtlNlRvUqQ=";
   };
 
   nativeBuildInputs = [ cmake gettext pkg-config wrapGAppsHook ];
   buildInputs =
-    [ glib
+    [
+      alsaLib
+      glib
```

Comment:
https://github.com/xournalpp/xournalpp/blob/release-1.2/CHANGELOG.md
Gtksourceview4 should be added as well, for improved latex editing.

Reply:
Done, and tested that it works